client/i18n/locales/chinese-traditional/intro.json

```
@@ -376,15 +376,25 @@
   "relational-databases": {
     "title": "關係型數據庫",
     "intro": [
+<<<<<<< HEAD
       "對於這些課程，你將使用真正的開發者工具和軟件，包括 VS Code、PostgreSQL 和 Linux / Unix 命令行，來完成交互式教程和構建項目。"
+=======
+      "For these courses, you will use real developer tools and software including VS Code, PostgreSQL, and the Linux / Unix command line to complete interactive tutorials and build projects."
```

```suggestion
      "For these courses, you will use real developer tools and software including Visual Studio Code, PostgreSQL, and the Linux/Unix command line to complete interactive tutorials and build projects."
```
For clarity and formatting's sake, I would write out the full name of VSCode and remove the spaces in between the slashes in between Linux and Unix.